src/WebJobs.Script.WebHost/Management/InstanceManager.cs

```diff
@@ -80,21 +84,47 @@ public async Task<string> ValidateContext(HostAssignmentContext assignmentContex
         {
             _logger.LogInformation($"Validating host assignment context (SiteId: {assignmentContext.SiteId}, SiteName: '{assignmentContext.SiteName}')");
 
+            string error = null;
+
+            try
+            {
+                await ValidateContextInternal(assignmentContext);
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e.ToString());
```

Comment:
LogError(e) - no need to ToString this yourself.

Reply:
Fixed